hooks/600-no-motd.chroot

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+set -eux
+
+systemctl mask motd-news.service
```

Comment:
afaik this will create more files in /etc ? Could we instead just `rm -rf /etc/system/timers.target.wants` ?

Reply:
Hmm, perhaps. I really think we need to look into the systemd preset system. As I mentioned earlier this would let us to control this *without* touching /etc.